android/modules/platform/src/java/ti/modules/titanium/platform/DisplayCapsProxy.java

```diff
@@ -75,10 +77,8 @@ public String getDensity() {
 				return "xxxhigh";
 			case 640:
 				return "xxxhigh";
-			case DisplayMetrics.DENSITY_LOW :
-				return "low";
 			default :
-				return "medium";
+				return "unknown";
```

Comment:
I gave you bad advise. It should not return "unknown". Developers use this name to load images of an appropriate size based on the "density" name property.  So, this code needs to be change to use >= checks.  It should also *not* return "medium" for unknown densities either.

The code should be changed to something like this...
`
public String getDensity() {
	synchronized(dm) {
		getDisplay().getMetrics(dm);
		int dpi = dm.densityDpi;
		if (dpi >= 560) {        // DisplayMetrics.DENSITY_560
			return "xxxhigh";
		} else if (dpi >= 400) { // DisplayMetrics.DENSITY_400
			return "xxhigh";
		} else if (dpi >= 280) { // DisplayMetrics.DENSITY_280
			return "xhigh";
		} else if (dpi >= DisplayMetrics.DENSITY_HIGH) {
			return "high";
		} else if (dpi >= DisplayMetrics.DENSITY_TV) {
			return "tvdpi";
		} else if (dpi >= DisplayMetrics.DENSITY_MEDIUM) {
			return "medium";
		}
		return "low";
	}
}
`